### PR TITLE
Symfony TreeBuilder deprecation

### PR DESCRIPTION
**Pull request by DemigodCode**
Since Symfony 3.0 TreeBuilder requires the root name in the constructor.
Symfony 4.2 is marking this behaviour as deprecated and will be removed in 5.0.

### CODE REVIEW

**Comment by florianeckerstorfer** [comment]
@DemigodCode Thanks for the PR, could you fix the tests?

**Comment by DemigodCode** [comment]
Okay, that was really tricky. I had to change some versions. Because symfony 4.2 is PHP7 and laravell requires symfony/config which has to be upgraded to 4.2 it is not possible to support laravell, Silex or symfony versions which are using <PHP7.

**Comment by DemigodCode** [comment]
Fixed all issues. But appveyor is failing, because it doesn't find php and composer. Don't know what to do.

File: .appveyor.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ install:
   - ps: Set-Service wuauserv -StartupType Manual
   - cinst -y php composer
   - refreshenv
-  - cd c:\tools\php72
+  - cd c:\tools\php73
```
Comment (DemigodCode) [inline .appveyor.yml]: PHP7.3 released yesterday

File: tests/Bridge/Latte/SlugifyHelperTest.php [hunk 2]
```diff
@@ -24,6 +24,10 @@ protected function setUp()
         $this->helper = new SlugifyHelper($this->slugify);
     }
 
+    protected function tearDown() {
+        m::close();
```
Comment (DemigodCode) [inline tests/Bridge/Latte/SlugifyHelperTest.php]: Mockery update needs close() after each test

File: tests/Bridge/Nette/SlugifyExtensionTest.php [hunk 3]
```diff
@@ -25,6 +25,7 @@ protected function setUp()
 
     /**
      * @test
+     * @doesNotPerformAssertions
```
Comment (DemigodCode) [inline tests/Bridge/Nette/SlugifyExtensionTest.php]: PHPUnit needs this annotation, if no assertions are made in test

**Comment by DemigodCode** [comment]
Hey, what do you think about removing the symfony dependency? I would build a bundle for symfony which registers the slugify as service and configures it? That might be the better solution, i think.

**Comment by kubawerlos** [comment]
Big :-1: for all the BC breaks to solve a vendor's deprecation, https://github.com/cocur/slugify/pull/220 is much better.

**Comment by florianeckerstorfer** [comment]
Fixed by #220

**Pull request closed without merging**